### DISCUSSION TITLE
Dynamic route for Newbie

### DISCUSSION THREAD

**Discussion by Trongnguyen-IT** [oqtane/oqtane.framework#4531] [Q&A]
Hi team,
Hope you are doing well!
I am building the web application with Oqtane. I have News page and Newsdetails page. But I don't know how to archive click to News item in list News and navigate to Newsdetails page. Could you please advise for this case?
Thank you!

**Comment by ADefWebserver** [comment]
This book may be helpful: [Custom Blazor Oqtane Modules Succinctly](https://www.syncfusion.com/succinctly-free-ebooks/custom-blazor-oqtane-modules-succinctly/create-the-help-desk-module)
It covers the code from tis project: https://github.com/ADefWebserver/Oqtane.SyncfusionHelpDesk

![Animation](https://user-images.githubusercontent.com/1857799/111890181-57772500-89a4-11eb-85c3-4e4cff6b1da8.gif)

**Comment by sbwalker** [comment]
@Trongnguyen-IT this question was fully answered here already: #4528 - was there something missing?

**Comment by Trongnguyen-IT** [comment]
the issue have been resolved . Thank you guys!
